students/piotrowski_stanislaw/lesson_01_basics/L1_IFN_06.py

```diff
@@ -0,0 +1,15 @@
+#https://snakify.org/lessons/integer_float_numbers/problems/car_route/
+#piotrsta
+
+from math import ceil
+
+print('Podaj mozliwy kilometraz na dzien: ')
+kmPerDay = int(input())
```

Comment:
W Pythonie jest zalecany inny styl nazewnictwa zmiennych. Zapoznaj się z PEP8 (link w sekcji 'Reference materials'), jest to przewodnik po coding convention w Pythonie.